source/sgp_mode/AnalysisTools.h

```diff
@@ -85,7 +118,7 @@ emp::vector<int> GetNecessaryInstructions(CPU org_cpu, size_t test_task_id) {
 
   if (can_do_task) {
 
-    for (size_t k = 0; k <= control_program.size() - 1; k++) {
+    for (int k = 0; k <= control_program.size() - 1; k++) {
```

Comment:
size_t is the more appropriate type, so you should go with the previous version of the code